compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -131,15 +167,32 @@ void RecordMinMax::profileData(const std::string &input_data_path)
   }
 
   auto minmax_map = _observer->minMaxData()->getMap();
-  (void *)minmax_map;
+  for (auto iter = minmax_map->begin(); iter != minmax_map->end(); ++iter)
+  {
+    auto node = iter->first;
+    auto minmax = iter->second;
+
+    // Default: Values are averaged without clipping (clip ratio = 0 %).
+    // TODO: Allow users to adjust clip ratios.
+    float min =
+        clipAndAverage(minmax.min_vector, /* top_clip_ratio */ 0.0, /* bottom_clip_ratio */ 0.0);
```

Comment:
this will get 0.0 ~ 0.0 ?

Reply:
Clip ratio is the ratio of clipped values. So, 0.0 means that we use all data. If top_clip_ratio is 1.0, top 1 % values will be discarded. If bottom_clip_ratio is 1.0, bottom 1% values will be discarded.